.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,8 +1,14 @@
 blank_issues_enabled: false
 contact_links:
-  - name: Ask a Question
+  - name: ❓ Ask a Question
     url: https://github.com/Lightning-AI/lightning/discussions/new
-    about: Ask and answer Lightning related questions
-  - name: 💬 Slack
+    about: Ask and answer Lightning related questions.
+  - name: 💬 Chat with us
     url: https://www.pytorchlightning.ai/community
-    about: Chat with our community
+    about: Live chat with experts, engineers, and users on our Slack community.
+  - name: 📖 Read the documentation
+    url: https://lightning.ai/lightning-docs/
+    about: Please consult the documentation before opening any issues!
+  - name: 🙋 Contact us about professional services
+    url: https://lightning.ai
```

Comment:
We should have a contact us option on `lightning.ai`. The standalone URL won't help the user out much.

Reply:
I agree, but there is only pop-up window now...
cc: @edenlightning @lantiga

Reply:
Me too, the other day I searched for several minutes for an email or contact form. There is nothing to be found xD